options/linux/generic/malloc.cpp

```diff
@@ -10,3 +10,7 @@ void *memalign(size_t, size_t) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
+int mallopt(int, int) {
+	return 1;
```

Comment:
Return zero instead. This didn't succeed (it isn't implemented). Funnily enough, errno isn't set by this function, which doesn't even let us mention that it's not implemented via ENOSYS. (Also, wtf? I've never seen this function in actual use)